cts/cts-schemas.in

```diff
@@ -589,27 +619,42 @@ test_suite() {
         esac
         shift
     done
+
+    # _ts_select contains a '@'-delimited list of test suite names from CLI
     _ts_select="${_ts_select}@"
+
+    # _ts_select_full contains a '@'-delimited list of test names(?)
```

Comment:
looks like the first is test names passed on stdin, and the second is test names passed on the command line. maybe also the first is suite names and the second individual tests?

Not for this PR, but we can drop the stdin ability and just take command line options

Reply:
On current main, no syntax seems to work for passing tests or test suites via stdin. It's always "Non-existing test(s)". Not sure if it got broken during recent refactors or if it's been broken for longer... not worth digging into if we're likely to drop stdin tests soon.

I meant to remove the `(?)` after verifying